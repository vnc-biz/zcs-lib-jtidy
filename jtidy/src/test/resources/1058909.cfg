tidy-mark: false
wrap: 0
clean: true
output-xhtml:true

tidy-mark= false
wrap= 0
clean= true
output-xhtml= true
