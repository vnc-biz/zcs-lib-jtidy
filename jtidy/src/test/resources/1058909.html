<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>[ #1058909 ] Certain sites causing null pointer Exceptions</title>
</head>
<body>
<div id="content">
<div class="blogbody">
<h2>NPE</h2> 
<div>
</div>
</body>
</html>