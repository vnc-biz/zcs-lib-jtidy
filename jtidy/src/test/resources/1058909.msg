<?xml version="1.0" encoding="ISO-8859-1"?>
<!-- expected messages for test 1058909-->
<messages>
  <message>
    <code>-1</code>
    <level>0</level>
    <line>0</line>
    <column>0</column>
    <text><![CDATA[Tidy (vers Sep 26, 2004) Parsing "InputStream"]]></text>
  </message>
  <message>
    <code>23</code>
    <level>2</level>
    <line>11</line>
    <column>-3</column>
    <text><![CDATA[trimming empty <div>]]></text>
  </message>
  <message>
    <code>6</code>
    <level>2</level>
    <line>13</line>
    <column>8</column>
    <text><![CDATA[missing </div>]]></text>
  </message>
  <message>
    <code>6</code>
    <level>2</level>
    <line>13</line>
    <column>8</column>
    <text><![CDATA[missing </div>]]></text>
  </message>
  <message>
    <code>110</code>
    <level>0</level>
    <line>1</line>
    <column>1</column>
    <text><![CDATA[InputStream: Doctype given is "-//W3C//DTD XHTML 1.0 Transitional//EN"]]></text>
  </message>
  <message>
    <code>111</code>
    <level>0</level>
    <line>1</line>
    <column>1</column>
    <text><![CDATA[InputStream: Document content looks like XHTML 1.0 Strict]]></text>
  </message>
  <message>
    <code>-1</code>
    <level>0</level>
    <line>0</line>
    <column>0</column>
    <text><![CDATA[3 warnings, no errors were found!]]></text>
  </message>
</messages>
