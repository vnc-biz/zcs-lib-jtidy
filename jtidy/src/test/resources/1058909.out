<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>[ #1058909 ] Certain sites causing null pointer Exceptions</title>
</head>
<body>
<div id="content" class="blogbody">
<h2>NPE</h2>
</div>
</body>
</html>
